src/js.rs

```diff
@@ -1,14 +1,30 @@
-use crate::{
-    ast::*, error::Error::UnsupportedFeature, fs::Utf8Writer, line_numbers::LineNumbers, Result,
-};
+mod statements;
+#[cfg(test)]
+mod tests;
+
+use crate::{ast::*, fs::Utf8Writer, line_numbers::LineNumbers, pretty::*, Result};
+use itertools::Itertools;
+
+const INDENT: isize = 4;
```

Comment:
Let's make this 2 as it seems to be more common.
```suggestion
const INDENT: isize = 2;
```